README.md

```diff
@@ -475,14 +258,14 @@ This is a variation on the materialised path database pattern.
 It allows Ancestry to fetch any relation (siblings,
```

Comment:
can we simplify by saying it is materialized path in a string with a `/` delimiter?